libs/domain/picking/warehouse-assignment/warehouse-assignment.component.ts

```diff
@@ -100,6 +100,11 @@ export class WarehouseAssignmentComponent extends LitElement {
   }
 
   protected renderLoading(): TemplateResult {
+    const locations = this.$locations();
+    if (locations?.length === 1) {
+      this.onSelect(locations[0].id);
```

Comment:
I have some concern regarding calling a method with the name `onSelect`. And another question is it a good idea to have this redirect in render function? Maybe we can make it earlier as long as we are not depending on user interaction?
Also it looks like it's worth to cover with tests.

Reply:
The render() method is dispatch every time the location property changes.
The firstupdate() lifecycle  could be a good approach, but when reached the locations$ property his still null.
Not sure there is any other alternative.
I will rename the method and update unit tests